services/ipp-values.go

```diff
@@ -0,0 +1,292 @@
+package services
+
+import "github.com/honeytrap/honeytrap/services/decoder"
+
+type ippValueType interface {
```

Comment:
if moved to seperate folder ipp, we can just use valueType instead of ippValueType